donkeycar/parts/imu.py

```diff
@@ -47,10 +94,18 @@ def shutdown(self):
 
 if __name__ == "__main__":
     iter = 0
-    p = Mpu6050()
+    import sys
+    sensor_type = SENSOR_MPU6050 
```

Comment:
main function allows testing only for MPU6050, shouldn't there be an extra argument to pass test for MPU9250?

Reply:
It defaults to mpu6050 and DLP=0 but you can pass two additional parameters: sensor type and DLP settings